p4runtime_sh/context.py

```diff
@@ -62,6 +62,10 @@ class P4RuntimeEntity(enum.Enum):
 class Context:
     def __init__(self):
         self.p4info = None
+        self.do_printing = True
+
+    def set_do_printing(self, do_printing):
+        self.do_printing = do_printing
```

Comment:
I'd rather keep this class as a P4Info-specific object.
I have 2 alternative solutions:
1) add a new global variable for this: https://github.com/p4lang/p4runtime-shell/blob/2603e13388adafcec77ede23658ca21f124d2502/p4runtime_sh/shell.py#L40-L41
2) define a new "global option": https://github.com/p4lang/p4runtime-shell/blob/main/p4runtime_sh/global_options.py

Honestly, option 2 may be the cleanest one, even though usage may be a bit different from what you have now:
```python
sh.setup(...)
global_options["verbose"] = False
```

I am also fine with adding the optional `setup` parameter, that will set the default value for `global_options["verbose"]`